apps/etterlatte-grunnlag/src/main/kotlin/grunnlag/omregning/OmregningDao.kt

```diff
@@ -0,0 +1,43 @@
+package no.nav.etterlatte.grunnlag.omregning
+
+import kotliquery.TransactionalSession
+import no.nav.etterlatte.libs.database.Transactions
+import no.nav.etterlatte.libs.database.hentListe
+import no.nav.etterlatte.libs.database.transaction
+import java.time.YearMonth
+import javax.sql.DataSource
+
+class OmregningDao(private val datasource: DataSource) : Transactions<OmregningDao> {
+    override fun <R> inTransaction(block: OmregningDao.(TransactionalSession) -> R): R =
+        datasource.transaction {
+            this.block(it)
+        }
+
+    fun hentSoekereFoedtIEnGittMaaned(
+        maaned: YearMonth,
+        tx: TransactionalSession? = null,
+    ): List<String> {
+        val sql =
+            """
+            select g.sak_id as sak_id from grunnlagshendelse g
```

Comment:
Her vil man vel potensielt få endel duplikater kanskje? 🤔

Reply:
Sant, denne kan jo absolutt vera select distinct 🤔

Reply:
Og bør man ta høyde for at opplysningen kan ha blitt oppdatert? Tenker f. eks at man kun ser på `opplysning_type = 'FOEDSELSDATO'` innslaget med høyest hendelsesnummer?

Reply:
Godt poeng. Oppdaterte no.